strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/maven/MavenArtifactControllerTest.java

```diff
@@ -279,7 +282,40 @@ private void resolveArtifact(String artifactPath,
         assertEquals("MD5 checksums did not match!", md5Local, md5Remote);
         assertEquals("SHA-1 checksums did not match!", sha1Local, sha1Remote);
     }
-
+    
+    @Test
+    public void testHeaderFetch()
+            throws Exception
+    {   
+        String artifactPath1 = "storages/storage0/act-releases-1/org/carlspring/strongbox/browse/foo-bar/2.4/foo-bar-2.4.jar";
+        
+       
+       
+        Headers headers1 = client.getHeaders(artifactPath1);
+       
+        
+        
+        String artifactPath2 = "storages/storage0/act-releases-1/org/carlspring/strongbox/browse/foo-bar/2.4/foo-bar-2.4.pom";
+        
+        logger.debug("Pausing now ");
```

Comment:
Is there anything actually sleeping here? (Like, for example, `Thread.sleep(1000)`)...?